js-weather/index.js

```diff
@@ -0,0 +1,62 @@
+// 天気のデータを取得
+function getWeatherData(cityData) {  
+  let targetCityName = cityData;
```

Comment:
```suggestion
  const targetCityName = cityData;
```
再代入しない変数は letではなくconstで宣言しましょう。
他のletもconstに置き換えられないか検討してください。